src/admin/WL_Metabox/WL_Metabox_Field_date.php

```diff
@@ -2,14 +2,37 @@
 
 
 class WL_Metabox_Field_date extends WL_Metabox_Field {
+	
+	/*
+	 * Attribute to distinguish between date formats, inferred from the schema property export type
+	 * 
+	 * @since 3.2.0
+	 */
+	private $date_format;
+	
+	/*
+	 * Boolean flag to decide if the calendar should include time or not
+	 * 
+	 * @since 3.2.0
+	 */
+	private $timepicker;
+	
+	public function __construct($args) {
+		
+		// Call parent constructor
+		parent::__construct($args);
+		
+		// Distinguish between date and datetime	
+		$this->date_format = 'Y/m/d';		// Default is date
+		if( isset( $this->raw_custom_field['export_type'] ) && 'xsd:datetime' === $this->raw_custom_field['export_type'] ) {
+			$this->date_format .= ' H:i';
+		}
+		$this->timepicker = ( strpos( $this->date_format, 'H:i') !== false );
+	}
     
     public function html_input( $date ) {
         
-        $pickerDate  = '';
-        if( !empty( $date ) ){
-            $pickerDate = date( 'Y/m/d H:i', strtotime( $date ) );
-        }
-        $pickerDate = esc_attr( $pickerDate );
+		$pickerDate = ( empty( $date ) ? '' :  esc_attr( date( $this->date_format, strtotime( $date ) ) ) );
```

Comment:
@pieroit `$pickerDate` does not respect the WP PHP Coding standards, rename to `$picker_date`